.changeset/thick-papayas-promise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@equinor/mad-components": patch
+---
+
+Added new component Switch, child component SmallSwitch and a SwitchScreen
```

Comment:
I would rephrase this to
> Added `Switch` and `Switch.Small` components

the code blocks helps users understand that these are actual jsx tags that can be used.
Also, the changeset is written for the mad components package, so we don't need to mention any SwitchScreens since that belongs to the chronicles app.